scripts/watch.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+process.env.FORCE_COLOR = 3;
+
+const shell = require("shelljs");
+const tools = require("./tools.js");
+
+const argv = require("yargs")
+  // Set up --help documentation.
+  // You can view these by running `npm run watch -- --help`.
+  .usage("Usage: npm watch -- [options]")
+  .example([
+    ["npm run watch", "(compile all components)"],
+    ["npm run watch -- pfe-card", "(compile one component)"],
+    ["npm run watch -- pfe-card pfe-band", "(compile multiple components)"],
+    ["npm run watch -- --build", "(compile assets before running watch)"],
+    ["npm run watch -- --storybook", "(watch storybook instance)"]
+  ])
+  .options({
+    build: {
+      default: false,
+      alias: "nb",
+      describe: "do not build the component(s) prior to running tests",
```

Comment:
I think this is the opposite of what we're doing. 🙃

Reply:
lol yes, yes you're right